stable/fairwinds-insights/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 2.4.1
```

Comment:
As this **adds** a new chart feature let's bump the minor version

Reply:
done